pkg/util/k8sutil/backup.go

```diff
@@ -45,6 +45,8 @@ const (
 	awsSecretVolName          = "secret-aws"
 	awsConfigVolName          = "config-aws"
 	fromDirMountDir           = "/mnt/backup/from"
+
+	backupDir = "/var/etcd-operator-backup/v1"
```

Comment:
i do not feel this should live inside k8sutil pkg. this should live in a pv specific file.